packages/messenger-widget/src/components/AddConversation/AddConversation.tsx

```diff
@@ -94,12 +94,23 @@ export default function AddConversation() {
         setErrorMsg('');
         setShowError(false);
         setTldName(e.target.value);
-        if (!ethers.utils.isValidName(e.target.value)) {
+
+        if (
+            !(
+                ethers.utils.isValidName(e.target.value) ||
+                isLuksoName(e.target.value)
+            )
+        ) {
```

Comment:
same logic, but might be easier to read:
```suggestion
        if (!ethers.utils.isValidName(e.target.value) && !isLuksoName(e.target.value)) {
```

Reply:
Good catch!